### PR TITLE
Add an actual `data-bs-touch="false"` example in the carousel docs

### PR DESCRIPTION
**Pull request by patrickhlauke**
this can also make it easier to actually check/debug any issues we may have with this data attribute

### CODE REVIEW

File: site/content/docs/5.0/components/carousel.md [hunk 1]
```diff
@@ -206,6 +206,34 @@ Add `data-bs-interval=""` to a `.carousel-item` to change the amount of time to
 </div>
 {{< /example >}}
 
+### Suppressing touch swiping
+
+By default, carousels support swiping left/right on touchscreen devices to move between slides. This can be explicitly suppressed using the `data-touch` attribute.
+
+{{< example >}}
+<div id="carouselExampleControlsNoTouching" class="carousel slide" data-bs-ride="carousel" data-touch="false">
```
Comment (rohit2sharma95) [inline site/content/docs/5.0/components/carousel.md:214]: `data-touch` does not contain bs namespace here, and attributes without bs namespace are ignored by Bootstrap.

Comment (patrickhlauke) [reply site/content/docs/5.0/components/carousel.md:214]: aha! perhaps that's all there is to it? (though that wouldn't explain the report from users saying it's also causing issues in 4.5)

Comment (rohit2sharma95) [reply site/content/docs/5.0/components/carousel.md:214]: NVM, I forgot it is for V4

**Comment by rohit2sharma95** [comment]
So it means the issue can not be reproduced in the main branch? (I can see that touch events are not working when `data-bs-touch` is set to false in the example)

**Comment by patrickhlauke** [comment]
> So it means the issue can not be reproduced in the main branch? (I can see that touch events are not working when `data-bs-touch` is set to false in the example)

correct, I edited/added this to the other issue about the supposed bug itself. still would like this as an addition to the docs regardless, as it makes it nice and obvious

**Review by rohit2sharma95 — APPROVED** [review]

File: site/content/docs/5.0/components/carousel.md [hunk 2]
```diff
@@ -297,15 +325,15 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
...
-      <p>On touch-enabled devices, when set to <code>"hover"</code>, cycling will pause on <code>touchend</code> (once the user finished interacting with the carousel) for two intervals, before automatically resuming. Note that this is in addition to the above mouse behavior.</p></td>
+      <td><code>'hover'</code></td>
+      <td><p>If set to <code>'hover'</code>, pauses the cycling of the carousel on <code>mouseenter</code> and resumes the cycling of the carousel on <code>mouseleave</code>. If set to <code>false</code>, hovering over the carousel won't pause it.</p>
+      <p>On touch-enabled devices, when set to <code>'hover'</code>, cycling will pause on <code>touchend</code> (once the user finished interacting with the carousel) for two intervals, before automatically resuming. Note that this is in addition to the above mouse behavior.</p></td>
     </tr>
     <tr>
-      <td><code>slide</code></td>
+      <td><code>ride</code></td>
```
Comment (XhmikosR) [inline site/content/docs/5.0/components/carousel.md:333]: Wonder why this was `slide` all this time...

Comment (patrickhlauke) [reply site/content/docs/5.0/components/carousel.md:333]: yes, either I missed something, or we've had a typo in the options all along...

(admittedly it's the first time I've properly looked at our carousels, but still)

Comment (XhmikosR) [reply site/content/docs/5.0/components/carousel.md:333]: Yeah, it looks like it's a typo AFAICT. Going to merge this later.

**Pull request merged**